Sources/FlexLayout.swift

```diff
@@ -31,7 +31,7 @@ public final class Flex {
     /**
      Flex items's UIView.
     */
-    public let view: UIView
+    private weak var view: UIView?
```

Comment:
We could just keep the setter private:

```
private (set) weak var view: UIView?
```

This way we could still access it as a weak reference.